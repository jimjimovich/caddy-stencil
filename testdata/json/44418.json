{"consolidated_weather":[{"id":6518532309254144,"weather_state_name":"Heavy Cloud","weather_state_abbr":"hc","wind_direction_compass":"SW","created":"2018-10-08T14:56:03.160418Z","applicable_date":"2018-10-08","min_temp":8.07,"max_temp":17.384999999999998,"the_temp":16.215,"wind_speed":7.408529880800127,"wind_direction":230.49923642046548,"air_pressure":1024.605,"humidity":71,"visibility":10.890151515151516,"predictability":71},{"id":4988721094983680,"weather_state_name":"Light Cloud","weather_state_abbr":"lc","wind_direction_compass":"SSW","created":"2018-10-08T14:56:03.758500Z","applicable_date":"2018-10-09","min_temp":9.8625,"max_temp":19.6925,"the_temp":18.27,"wind_speed":6.823458422713354,"wind_direction":197.62484448875344,"air_pressure":1021.76,"humidity":66,"visibility":11.833703670563906,"predictability":70},{"id":5343432880947200,"weather_state_name":"Light Cloud","weather_state_abbr":"lc","wind_direction_compass":"ESE","created":"2018-10-08T14:56:03.759557Z","applicable_date":"2018-10-10","min_temp":11.907499999999999,"max_temp":22.662499999999998,"the_temp":21.59,"wind_speed":7.782770120888014,"wind_direction":123.38929219496218,"air_pressure":1016.11,"humidity":69,"visibility":9.957473355603277,"predictability":70},{"id":5272476397338624,"weather_state_name":"Showers","weather_state_abbr":"s","wind_direction_compass":"S","created":"2018-10-08T14:56:03.672642Z","applicable_date":"2018-10-11","min_temp":14.2875,"max_temp":19.5975,"the_temp":20.439999999999998,"wind_speed":10.982928180269228,"wind_direction":173.1674391535682,"air_pressure":1007.5799999999999,"humidity":70,"visibility":15.213341585142766,"predictability":73},{"id":6396366527922176,"weather_state_name":"Heavy Cloud","weather_state_abbr":"hc","wind_direction_compass":"SSW","created":"2018-10-08T14:56:01.789339Z","applicable_date":"2018-10-12","min_temp":13.3575,"max_temp":19.57,"the_temp":19.12,"wind_speed":14.004277149262307,"wind_direction":197.62381562008204,"air_pressure":1014.54,"humidity":69,"visibility":12.995978485643839,"predictability":71},{"id":5118373239193600,"weather_state_name":"Heavy Cloud","weather_state_abbr":"hc","wind_direction_compass":"S","created":"2018-10-08T14:56:04.365990Z","applicable_date":"2018-10-13","min_temp":15.5825,"max_temp":21.1325,"the_temp":22.47,"wind_speed":11.002789708104666,"wind_direction":170.93681505881997,"air_pressure":1013.84,"humidity":67,"visibility":9.997862483098704,"predictability":71}],"time":"2018-10-08T17:55:15.400487+01:00","sun_rise":"2018-10-08T07:12:27.756857+01:00","sun_set":"2018-10-08T18:22:45.464351+01:00","timezone_name":"LMT","parent":{"title":"England","location_type":"Region / State / Province","woeid":24554868,"latt_long":"52.883560,-1.974060"},"sources":[{"title":"BBC","slug":"bbc","url":"http://www.bbc.co.uk/weather/","crawl_rate":180},{"title":"Forecast.io","slug":"forecast-io","url":"http://forecast.io/","crawl_rate":480},{"title":"HAMweather","slug":"hamweather","url":"http://www.hamweather.com/","crawl_rate":360},{"title":"Met Office","slug":"met-office","url":"http://www.metoffice.gov.uk/","crawl_rate":180},{"title":"OpenWeatherMap","slug":"openweathermap","url":"http://openweathermap.org/","crawl_rate":360},{"title":"Weather Underground","slug":"wunderground","url":"https://www.wunderground.com/?apiref=fc30dc3cd224e19b","crawl_rate":720},{"title":"World Weather Online","slug":"world-weather-online","url":"http://www.worldweatheronline.com/","crawl_rate":360},{"title":"Yahoo","slug":"yahoo","url":"http://weather.yahoo.com/","crawl_rate":180}],"title":"London","location_type":"City","woeid":44418,"latt_long":"51.506321,-0.12714","timezone":"Europe/London"}