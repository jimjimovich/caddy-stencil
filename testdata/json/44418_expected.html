<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>London</title>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.1/css/bulma.min.css" type="text/css" />
  </head>
  <body>
    <section class="section">
      <div class="container">
        
          <h1 class="title is-1">London Weather</h1>
          <div class="columns">
            
            <div class="column">
              <div class="card">
                <div class="card-content">
                  <p class="title is-3">10/08</p>
                  <figure class="image">
                    <img src="/static/img/weather/hc.svg" alt="Heavy Cloud">
                  </figure>
                  <div class="content has-text-centered">
                    <p class="title is-4">Heavy Cloud</p>
                    <p class="title is-4">8.1 - 17.4&deg; C</p>
                    <p class="title is-4">Humidity 71%</p>
                  </div>
                </div>
              </div>
            </div>
            
            <div class="column">
              <div class="card">
                <div class="card-content">
                  <p class="title is-3">10/09</p>
                  <figure class="image">
                    <img src="/static/img/weather/lc.svg" alt="Light Cloud">
                  </figure>
                  <div class="content has-text-centered">
                    <p class="title is-4">Light Cloud</p>
                    <p class="title is-4">9.9 - 19.7&deg; C</p>
                    <p class="title is-4">Humidity 66%</p>
                  </div>
                </div>
              </div>
            </div>
            
            <div class="column">
              <div class="card">
                <div class="card-content">
                  <p class="title is-3">10/10</p>
                  <figure class="image">
                    <img src="/static/img/weather/lc.svg" alt="Light Cloud">
                  </figure>
                  <div class="content has-text-centered">
                    <p class="title is-4">Light Cloud</p>
                    <p class="title is-4">11.9 - 22.7&deg; C</p>
                    <p class="title is-4">Humidity 69%</p>
                  </div>
                </div>
              </div>
            </div>
            
            <div class="column">
              <div class="card">
                <div class="card-content">
                  <p class="title is-3">10/11</p>
                  <figure class="image">
                    <img src="/static/img/weather/s.svg" alt="Showers">
                  </figure>
                  <div class="content has-text-centered">
                    <p class="title is-4">Showers</p>
                    <p class="title is-4">14.3 - 19.6&deg; C</p>
                    <p class="title is-4">Humidity 70%</p>
                  </div>
                </div>
              </div>
            </div>
            
            <div class="column">
              <div class="card">
                <div class="card-content">
                  <p class="title is-3">10/12</p>
                  <figure class="image">
                    <img src="/static/img/weather/hc.svg" alt="Heavy Cloud">
                  </figure>
                  <div class="content has-text-centered">
                    <p class="title is-4">Heavy Cloud</p>
                    <p class="title is-4">13.4 - 19.6&deg; C</p>
                    <p class="title is-4">Humidity 69%</p>
                  </div>
                </div>
              </div>
            </div>
            
            <div class="column">
              <div class="card">
                <div class="card-content">
                  <p class="title is-3">10/13</p>
                  <figure class="image">
                    <img src="/static/img/weather/hc.svg" alt="Heavy Cloud">
                  </figure>
                  <div class="content has-text-centered">
                    <p class="title is-4">Heavy Cloud</p>
                    <p class="title is-4">15.6 - 21.1&deg; C</p>
                    <p class="title is-4">Humidity 67%</p>
                  </div>
                </div>
              </div>
            </div>
            
          </div>
        
        <div class="columns">
          <div class="column">
            <form action="/api/location/search/">
              <div class="field has-addons">
                <div class="control">
                  <input class="input" type="text" name="query" placeholder="Find a city">
                </div>
                <div class="control">
                  <input class="button is-info" type="submit" value="Search"/>
                </div>
              </div>
            </form>
          </div>
          <div class="column">
            <p class="has-text-right">data provided by <a href="https://www.metaweather.com/">metaweather.com</a></p>
          </div>
        </div>
      </div>
    </section>
    .Doc.body should be empty
  </body>
</html>